File: Functions/6.cpp
#include <bits/stdc++.h>
using namespace std;

bool prime_or_not(int n) {
    if (n < 2){
        return false;
    }
    for (int i = 2; i <= sqrt(n); i++) {
        if (n % i == 0) {
            return false;
        }
    }
    return true;
}

void prime_until_n(int n) {
    cout << "Prime Numbers until " << n << ": " << endl;
    for (int i = 2; i <= n; i++) {
        if (prime_or_not(i)) {
            cout << i << " ";
        }
    }
    cout << endl;
}

int main() {
    int n;
    cout << "Enter a Number: ";
    cin >> n;
    prime_until_n(n);
}

File: Functions/6.py
def prime_or_not(n):
    for i in range(2,int(n**0.5) + 1):
        if(n%i == 0):
            return False
            break
    return True

def print_prime_until_n(n):
    if(n==1):
        print(" NONE ")
        return
    for i in range(2,n+1):
        if(prime_or_not(i)==True):
            print(i,end = " ")

n = int(input("Enter a Number: "))
print("Prime Numbers until ",n," : ")
print_prime_until_n(n)